### DISCUSSION TITLE
Run section

### DISCUSSION THREAD

**Discussion by Tatlim12** [Acode-Foundation/Acode#1036] [Q&A]
### Check for existing issues

- [X] Completed

### Describe the feature

I  don't   have  run  section

**Comment by Tatlim12** [comment]
![Uploading Screenshot_20240919_223952_Acode.jpg…]()

**Comment by Tatlim12** [comment]
![Screenshot_20240919_223952_Acode](https://github.com/user-attachments/assets/6e4cc543-0e0a-49f5-ad60-8670df66f33b)

**Comment by bajrangCoder** [comment]
Which programming language you want to run ?
If html, js then open that file , and you will get play ▶️ icon to run that.
For other programming language check plugins
